### ISSUE TITLE
Add offset option for smooth scroll to hashlinks

### ISSUE THREAD

**Issue by KittenCodes** [soflyy/oxygen-bugs-and-features#305]
When anchor links are used to scroll the page to the target ID, the target element is cut off if the sticky header is active.

Replicated here: http://energetic-orangutan.w6.wpsandbox.pro?upass=7eYjPP1ae61C
Page: http://energetic-orangutan.w6.wpsandbox.pro/product-tour/

Click the "SCROLL TO DIV" button and you'll see that the sticky header covers the div element that you've scrolled to.

Suggested by @wplit

**Comment by roman02** [comment]
There's no workaround for this. This deficiency ruins the navigation.

**Comment by KittenCodes** [comment]
@roman02 

There's a workaround using jQuery. Here's an example that can be used on the JavaScript tab of Oxygen's Code Block element:

```js
var $header = jQuery('#_header-20-15');

jQuery('a[href^="#"]').click(function(e){
	e.preventDefault();
	jQuery('html,body').animate({scrollTop:jQuery(this.hash).offset().top - $header.height()}, 750);
});
```

You need to replace _header-20-15 with the ID of the element that you want to offset the scroll by.

**Comment by porg** [comment]
I propose  to close this one as a dupe of #133
And over at #133 I will propose to close the issue as it was implemented in in [3.8 according to your news](https://oxygenbuilder.com/2021/06/11/oxygen-3-8-now-available/).

**Comment by KittenCodes** [comment]
Thanks @porg. Smooth scroll offset added in v3.8.
